packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -27,12 +31,19 @@ export interface ChatMessageActionFlyoutProps {
   onRemoveClick?: () => void;
   onDismiss: () => void;
   messageReadByCount?: number;
+  messageReadByNames?: { id: string; name: string }[];
```

Comment:
`Names` suffix is a misnomer here as it's an object with name and id, not just a name.
Consider just `messageReadBy` here

Reply:
changed name to messageReadBy